.github/workflows/generate.yml

```diff
@@ -33,24 +33,32 @@ jobs:
           go-version: '^1.17.3'
 
       - name: Generate CPE database
-        run: cd fleet && go mod download && go run -tags fts5 cmd/cpe/generate.go && cat ./etagenv >> $GITHUB_ENV
+        run: |
+          cd fleet
+          go mod download
+          go run -tags fts5 cmd/cpe/generate.go
+
+      - name: Current date
```

Comment:
Use the current date for releases. If multiple releases are created on the same day, then github sorts in descending order by name. Using the etag (a hash) results in random order.